### PR TITLE
docs: amend resource type in example to deployment

### PR DESCRIPTION
**Pull request by johnboy-leeds**
## Pull request description 
Amend the resource type in the example to `deployment` so it aligns with the conditions.

## Checklist (choose whats happened)

- [ ] breaking change! (describe)
- [ ] tested locally
- [ ] tested on cluster
- [ ] added new dependencies
- [x] updated the docs
- [ ] added a test

## Breaking changes

-

## Changes

-

## Fixes

-

### CODE REVIEW

File: docs/docs/using-testkube/triggers.md [hunk 1]
```diff
@@ -95,7 +95,7 @@ metadata:
   name: testtrigger-example
   namespace: default
 spec:
-  resource: pod
+  resource: deployment
```
Comment (vsukhin) [inline docs/docs/using-testkube/triggers.md:98]: please fix line 89 as well

**Review by vsukhin — COMMENTED** [review]
one comment

**Review by vsukhin — APPROVED** [review]
looks good!

**Review by nicufk — APPROVED** [review]

**Review by jfermi — APPROVED** [review]
Approved

**Pull request merged**